benchmark/single-source/Breadcrumbs.swift

```diff
@@ -113,21 +113,17 @@ class BenchmarkBase {
     self.inputString = ""
   }
 
-  final func run(iterations: Int) {
-    for _ in 0 ..< iterations {
-      self.run()
-    }
+  func run(iterations: Int) {
+    fatalError("unimplemented abstract method")
   }
 
-  func run() {}
-
   var info: BenchmarkInfo {
     return BenchmarkInfo(
       name: self.label,
-      runFunction: self.run(iterations:),
+      runFunction: { self.run(iterations: $0) },
```

Comment:
Is the closure necessary? The original form, directly referencing the method, worked fine, right?

Reply:
Yeah, there isn't any meaningful difference; I just forgot to revert this chunk before committing. It's a harmless change, though, so I'll leave it in.